### ISSUE TITLE
Event rendering when excessive minTime/maxTime

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#2530]
Originally reported on Google Code with ID 2265

```
My project StudySauce.com uses fullcalendar in a beautiful way.  However, we need the
ability to set the time range from 8 AM to 2 AM.  Fortunately, maxTime and minTime
can be used.  The time slots and general appearance work perfectly!
We ran in to issues with some events that occur between 12 AM and 2 AM, they were being
rendered at the minTime and the height was set to 0.  I also saw cases where the event
would start to render properly, but the next day it overflows in to wouldn't render.
 I have implemented a fix for this and would like to see it make it in to the next
version.  I am running version 2.1.0-beta1 so this may be fixed already.  Please compare
this code with the latest version.
The only change is to the computeTimeTop() function.  This subtracts the slot location
from 48 slots.  Attached is a diff for version 2.1.0-beta1, but it needs to be integrated
and retested in the latest 2.1 release.

// constrain. because minTime/maxTime might be customized
if(slatCoverage < 0)
{
      slatCoverage = (24 * 3600000/this.slotDuration * this.slotDuration + time - this.minTime)
/ this.slotDuration;
      slatCoverage = Math.max(0, slatCoverage);
      slatCoverage = Math.min(this.slatEls.length, slatCoverage);

      slatIndex = Math.floor(slatCoverage); // an integer index of the furthest whole
slot

}
else
{
      slatCoverage = Math.max(0, slatCoverage);
      slatCoverage = Math.min(this.slatEls.length, slatCoverage);

      slatIndex = Math.floor(slatCoverage); // an integer index of the furthest whole
slot
}
slatRemainder = slatCoverage - slatIndex;

```

Reported by `bjcullinan` on 2014-08-29 18:23:59

<hr>
- _Attachment: fullcalendar.png<br>![fullcalendar.png](https://storage.googleapis.com/google-code-attachments/fullcalendar/issue-2265/comment-0/fullcalendar.png)_
- _Attachment: [fullcalendar.js](https://storage.googleapis.com/google-code-attachments/fullcalendar/issue-2265/comment-0/fullcalendar.js)_
- _Attachment: [fullcalendar-2.1.0-beta1-2014-08-29.patch](https://storage.googleapis.com/google-code-attachments/fullcalendar/issue-2265/comment-0/fullcalendar-2.1.0-beta1-2014-08-29.patch)_

Imported with **2** stars.

**Comment by arshaw** [comment]
updated fiddle: http://jsbin.com/dajamekehe/1/edit?js,output

**Comment by arshaw** [comment]
a new simplified JSBin: http://jsbin.com/qajipewoxu/edit?js,output
one might expect the event to show up in the Monday column

**Comment by arshaw** [comment]
related to #3186

**Comment by arshaw** [comment]
updated js fiddle: http://jsbin.com/gixugaqoje/1/edit?js,output

**Comment by arshaw** [comment]
This should be fixed in **v3.3.0**
Blog post: https://fullcalendar.io/blog/2017/03/fullcalendar-33-and-scheduler-16/
